releasenotes/notes/0.17/prepare-0.17-2ab9429b69e1d25c.yaml

```diff
@@ -0,0 +1,28 @@
+---
+prelude: |
+    The Qiskit Terra 0.17.0 includes many new features and bug fixes. The major
+    new feature for this release is the introduction of the
+    :mod:`qiskit.algorithms` and :mod:`qiskit.opflow` modules which were
+    migrated and adapted from the :mod:`qiskit.aqua` project.
+features:
+  - |
+    A new module :mod:`qiskit.algorithms` has been introduced. This module
+    contains functionality equivalent to what has previously been
+    provided by the :mod:`qiskit.aqua.algorithms` module (which is now
+    deprecated) and provides the building blocks for constructing quantum
+    algorithms. For details on migrating from ``qiskit-aqua`` to this new
+    module please refer to the migration guide "INSERT LINK HERE"
```

Comment:
Is this link available?

Reply:
I tested this locally and adding a ref link to a nonexistent tag will raise a warning (which would block the docs job here). So I think for now it is best to just leave this as is and it's something I'll fix manually when I integrate the release notes into the metapackage docs.

Reply:
I will also follow up post release with an http link to the hosted migration guide.